assets/config/dev.json

```diff
@@ -0,0 +1,3 @@
+{
+    "baseUrl": "https://madev.creaf.cat"
```

Comment:
This is no longer the URL for the dev server. Change it to: `https://webdev.mosquitoalert.com`

Reply:
You're right, I pulled the branch from main instead of v33 so I mixed it by accident